packages/react/src/components/UIShell/HeaderMenu.js

```diff
@@ -185,7 +185,7 @@ class HeaderMenu extends React.Component {
           ref={this.handleMenuButtonRef}
           role="menuitem"
           tabIndex={0}
-          ariaLabel={ariaLabel}>
+          "aria-label"={ariaLabel}>
```

Comment:
```suggestion
          aria-label={ariaLabel}>
```

Could you try this? Thanks!

Reply:
It fails. it made `aria - label` an expression and is not able to compile.

Reply:
Could you try below with `aria-label` (without quotes) change:

```shell
> cd /path/to/carbon
> yarn install
> npx lerna run build
> cd packages/react
> yarn storybook
```

Above works for me. [React ARIA example code](https://reactjs.org/docs/accessibility.html#wai-aria) does not have quotes. Also, if you search for `aria-label` in our React source code, you'll find a bunch of results. Everything is without quotes:

<details>
 <summary><code>aria-label</code> usage</summary>

```
Searching 267 files for "aria-label="

\\path\to\carbon\packages\react\src\components\AccordionItem\AccordionItem.js:
  135            <ChevronRight16
  136              className={`${prefix}--accordion__arrow`}
  137:             aria-label={iconDescription}
  138            />
  139            <div className={`${prefix}--accordion__title`}>{title}</div>

\\path\to\carbon\packages\react\src\components\Button\Button.js:
   51    const buttonImage = !ButtonImageElement ? null : (
   52      <ButtonImageElement
   53:       aria-label={iconDescription}
   54        className={`${prefix}--btn__icon`}
   55        aria-hidden={true}

\\path\to\carbon\packages\react\src\components\CodeSnippet\CodeSnippet.js:
  153          </span>
  154          <ChevronDown16
  155:           aria-label={expandCodeBtnText}
  156            className={`${prefix}--icon-chevron--down ${prefix}--snippet__icon`}
  157            name="chevron--down"
  ...
  166          tabIndex={0}
  167          className={`${prefix}--snippet-container`}
  168:         aria-label={ariaLabel || copyLabel || 'code-snippet'}>
  169          <code>
  170            <pre
  ...
  191            {...other}
  192            onClick={onClick}
  193:           aria-label={copyLabel || ariaLabel}
  194            aria-describedby={this.uid}
  195            className={codeSnippetClasses}

\\path\to\carbon\packages\react\src\components\ComboBox\ComboBox.js:
  313                <input
  314                  className={`${prefix}--text-input`}
  315:                 aria-label={ariaLabel}
  316                  aria-controls={`${id}__menu`}
  317                  aria-autocomplete="list"
  ...
  337              </ListBox.Field>
  338              {isOpen && (
  339:               <ListBox.Menu aria-label={ariaLabel} id={id}>
  340                  {this.filterItems(items, itemToString, inputValue).map(
  341                    (item, index) => (

\\path\to\carbon\packages\react\src\components\ComposedModal\ComposedModal.js:
  352            title={iconDescription}
  353            type="button">
  354:           <Close20 aria-label={iconDescription} className={closeIconClass} />
  355          </button>
  356        </div>

\\path\to\carbon\packages\react\src\components\CopyButton\CopyButton.js:
   96          <Copy16
   97            className={`${prefix}--snippet__icon`}
   98:           aria-label={iconDescription}
   99          />
  100          <div className={feedbackClassNames} data-feedback={feedback} />

\\path\to\carbon\packages\react\src\components\DataTable\TableExpandHeader.js:
   36            onClick={onExpand}
   37            title={expandIconDescription}
   38:           aria-label={ariaLabel}>
   39            <ChevronRight16
   40              className={`${prefix}--table-expand__svg`}
   41:             aria-label={expandIconDescription}
   42            />
   43          </button>

\\path\to\carbon\packages\react\src\components\DataTable\TableExpandRow.js:
   46            onClick={onExpand}
   47            title={expandIconDescription}
   48:           aria-label={ariaLabel}>
   49            <ChevronRight16
   50              className={`${prefix}--table-expand__svg`}
   51:             aria-label={expandIconDescription}
   52            />
   53          </button>

\\path\to\carbon\packages\react\src\components\DataTable\TableHeader.js:
   80          <Arrow
   81            className={`${prefix}--table-sort__icon`}
   82:           aria-label={t('carbon.table.header.icon.description', {
   83              header: children,
   84              sortDirection,
   ..
   89          <Arrows
   90            className={`${prefix}--table-sort__icon-unsorted`}
   91:           aria-label={t('carbon.table.header.icon.description', {
   92              header: children,
   93              sortDirection,

\\path\to\carbon\packages\react\src\components\DataTable\TableToolbar.js:
   12  const TableToolbar = ({ children }) => (
   13    <section
   14:     aria-label="data table toolbar"
   15      className={`${prefix}--table-toolbar`}>
   16      {children}

\\path\to\carbon\packages\react\src\components\DatePickerInput\DatePickerInput.js:
   90          <Calendar16
   91            className={`${prefix}--date-picker__icon`}
   92:           aria-label={iconDescription}
   93            onClick={openCalendar}
   94            role="img">

\\path\to\carbon\packages\react\src\components\Dropdown\Dropdown.js:
  256                </ListBox.Field>
  257                {isOpen && (
  258:                 <ListBox.Menu aria-label={ariaLabel} id={id}>
  259                    {items.map((item, index) => (
  260                      <ListBox.MenuItem

\\path\to\carbon\packages\react\src\components\FileUploader\FileUploader.js:
  238          <CloseFilled16
  239            className={`${prefix}--file-close`}
  240:           aria-label={iconDescription}
  241            style={style}
  242            {...other}>
  ...
  248          <CheckmarkFilled16
  249            className={`${prefix}--file-complete`}
  250:           aria-label={iconDescription}
  251            style={style}
  252            {...other}>

\\path\to\carbon\packages\react\src\components\Icon\Icon.js:
  144  
  145    return (
  146:     <svg {...props} aria-label={description}>
  147        <title>
  148          {typeof iconTitle === 'undefined' ? description : iconTitle}

\\path\to\carbon\packages\react\src\components\InlineCheckbox\InlineCheckbox.js:
  133              htmlFor={id}
  134              className={`${prefix}--checkbox-label`}
  135:             aria-label={ariaLabel}
  136              title={title}
  137            />

\\path\to\carbon\packages\react\src\components\ListBox\ListBoxMenuIcon.js:
   36    return (
   37      <div className={className} role="button">
   38:       <ChevronDown16 name="chevron--down" aria-label={description}>
   39          <title>{description}</title>
   40        </ChevronDown16>

\\path\to\carbon\packages\react\src\components\Loading\Loading.js:
   70        <div
   71          {...other}
   72:         aria-label={description}
   73          aria-live={active ? 'assertive' : 'off'}
   74          className={loadingClasses}>

\\path\to\carbon\packages\react\src\components\Modal\Modal.js:
  284          onClick={onRequestClose}
  285          title={iconDescription}
  286:         aria-label={iconDescription}
  287          ref={this.button}>
  288          <Close20
  289:           aria-label={iconDescription}
  290            className={`${prefix}--modal-close__icon`}
  291          />
  ...
  298          role="dialog"
  299          className={`${prefix}--modal-container`}
  300:         aria-label={modalAriaLabel}
  301          aria-modal="true">
  302          <div className={`${prefix}--modal-header`}>

\\path\to\carbon\packages\react\src\components\MultiSelect\FilterableMultiSelect.js:
  380                    </ListBox.Field>
  381                    {isOpen && (
  382:                     <ListBox.Menu aria-label={ariaLabel} id={id}>
  383                        {sortItems(
  384                          filterItems(items, { itemToString, inputValue }),

\\path\to\carbon\packages\react\src\components\MultiSelect\MultiSelect.js:
  330                    </ListBox.Field>
  331                    {isOpen && (
  332:                     <ListBox.Menu aria-label={ariaLabel} id={id}>
  333                        {sortItems(items, {
  334                          selectedItems: {

\\path\to\carbon\packages\react\src\components\Notification\Notification.js:
   97        if (Object(IconTag) === IconTag) {
   98          return (
   99:           <IconTag aria-label={ariaLabel} className={iconClasses} name={name} />
  100          );
  101        }
  ...
  107          {...other}
  108          type={type}
  109:         aria-label={iconDescription}
  110          title={iconDescription}
  111          className={buttonClasses}>

\\path\to\carbon\packages\react\src\components\NumberInput\NumberInput.js:
  307                        onClick={evt => this.handleArrowClick(evt, 'down')}
  308                        title={decrementNumLabel}
  309:                       aria-label={decrementNumLabel || iconDescription}
  310                        aria-live="polite"
  311                        aria-atomic="true">
  ...
  324                        onClick={evt => this.handleArrowClick(evt, 'up')}
  325                        title={incrementNumLabel}
  326:                       aria-label={incrementNumLabel || iconDescription}
  327                        aria-live="polite"
  328                        aria-atomic="true">
  ...
  357                        onClick={evt => this.handleArrowClick(evt, 'up')}
  358                        title={incrementNumLabel || iconDescription}
  359:                       aria-label={incrementNumLabel || iconDescription}
  360                        aria-live="polite"
  361                        aria-atomic="true">
  ...
  367                        onClick={evt => this.handleArrowClick(evt, 'down')}
  368                        title={decrementNumLabel || iconDescription}
  369:                       aria-label={decrementNumLabel || iconDescription}
  370                        aria-live="polite"
  371                        aria-atomic="true">

\\path\to\carbon\packages\react\src\components\OverflowMenu\OverflowMenu.js:
  536            onBlur={this.handleBlur}
  537            onClick={this.handleClick}
  538:           aria-label={ariaLabel}
  539            id={id}
  540            tabIndex={tabIndex}

\\path\to\carbon\packages\react\src\components\Pagination\Pagination.js:
  328              className={backButtonClasses}
  329              onClick={this.decrementPage}
  330:             aria-label={backwardText}
  331              disabled={backButtonDisabled}>
  332              <CaretLeft24 />
  ...
  334            <button
  335              className={forwardButtonClasses}
  336:             aria-label={forwardText}
  337              onClick={this.incrementPage}
  338              disabled={forwardButtonDisabled || isLastPage}>

\\path\to\carbon\packages\react\src\components\ProgressIndicator\ProgressIndicator.js:
   53  
   54    const completeSvg = complete && (
   55:     <CheckmarkOutline16 aria-label={description} role="img">
   56        <title>{description}</title>
   57      </CheckmarkOutline16>

\\path\to\carbon\packages\react\src\components\RadioButton\RadioButton.js:
  133            htmlFor={this.uid}
  134            className={`${prefix}--radio-button__label`}
  135:           aria-label={labelText}>
  136            <span className={`${prefix}--radio-button__appearance`} />
  137            <span className={innerLabelClasses}>{labelText}</span>

\\path\to\carbon\packages\react\src\components\RadioTile\RadioTile.js:
  109            onKeyDown={this.handleKeyDown}>
  110            <div className={`${prefix}--tile__checkmark`}>
  111:             <CheckmarkFilled aria-label={iconDescription}>
  112                {iconDescription && <title>{iconDescription}</title>}
  113              </CheckmarkFilled>

\\path\to\carbon\packages\react\src\components\Search\Search.js:
  149          <Search16
  150            className={`${prefix}--search-magnifier`}
  151:           aria-label={labelText}
  152            role="img"
  153          />
  ...
  170            onClick={this.clearInput}
  171            type="button"
  172:           aria-label={closeButtonLabelText}>
  173:           <CloseIconX aria-label={closeButtonLabelText} role="img" />
  174          </button>
  175        </div>

\\path\to\carbon\packages\react\src\components\SearchFilterButton\SearchFilterButton.js:
   20      className={`${prefix}--search-button`}
   21      type="button"
   22:     aria-label={labelText}
   23      title={labelText}
   24      {...other}>
   25      <Filter16
   26        className={`${prefix}--search-filter`}
   27:       aria-label={iconDescription}
   28      />
   29    </button>

\\path\to\carbon\packages\react\src\components\SearchLayoutButton\SearchLayoutButton.js:
   84            <ListBulleted16
   85              className={`${prefix}--search-view`}
   86:             aria-label={iconDescriptionList}
   87            />
   88          );
   ..
   91          <Grid16
   92            className={`${prefix}--search-view`}
   93:           aria-label={iconDescriptionGrid}
   94          />
   95        );
   ..
  100          type="button"
  101          onClick={this.toggleLayout}
  102:         aria-label={labelText}
  103          title={labelText}>
  104          <div className={`${prefix}--search__toggle-layout__container`}>

\\path\to\carbon\packages\react\src\components\Select\Select.js:
   77          <ChevronDownGlyph
   78            className={`${prefix}--select__arrow`}
   79:           aria-label={iconDescription}>
   80            <title>{iconDescription}</title>
   81          </ChevronDownGlyph>

\\path\to\carbon\packages\react\src\components\Slider\Slider.js:
  470                onChange={this.handleChange}
  471                labelText=""
  472:               aria-label={ariaLabelInput}
  473                disabled={disabled}
  474              />

\\path\to\carbon\packages\react\src\components\StructuredList\StructuredList.js:
   64  
   65      return (
   66:       <section className={classes} {...other} aria-label={ariaLabel}>
   67          {children}
   68        </section>

\\path\to\carbon\packages\react\src\components\Tabs\Tabs.js:
  257            <div
  258              role="listbox"
  259:             aria-label={ariaLabel}
  260              tabIndex={0}
  261              className={`${prefix}--tabs-trigger`}

\\path\to\carbon\packages\react\src\components\Tag\Tag.js:
   40        {...other}>
   41        {children !== null && children !== undefined ? children : TYPES[type]}
   42:       <Close16 aria-label="Clear filter" />
   43      </span>
   44    ) : (

\\path\to\carbon\packages\react\src\components\TextInput\ControlledPasswordInput.js:
   98          <button
   99            className={passwordVisibilityToggleButtonClasses}
  100:           aria-label={
  101              alt ||
  102              `${

\\path\to\carbon\packages\react\src\components\TextInput\PasswordInput.js:
  101          <button
  102            className={passwordVisibilityToggleButtonClasses}
  103:           aria-label={alt || `${passwordIsVisible ? 'Hide' : 'Show'} password`}
  104            onClick={this.togglePasswordVisibility}>
  105            {passwordVisibilityIcon}

\\path\to\carbon\packages\react\src\components\Tile\Tile.js:
  294            onKeyDown={this.handleKeyDown}>
  295            <div className={`${prefix}--tile__checkmark`}>
  296:             <CheckmarkFilled aria-label={iconDescription}>
  297                {iconDescription && <title>{iconDescription}</title>}
  298              </CheckmarkFilled>
  ...
  473            <ChevronDown16
  474              id={buttonId}
  475:             aria-label={
  476                this.state.expanded ? tileExpandedIconText : tileCollapsedIconText
  477              }

\\path\to\carbon\packages\react\src\components\TimePickerSelect\TimePickerSelect.js:
  111          <ChevronDownGlyph
  112            className={`${prefix}--select__arrow`}
  113:           aria-label={iconDescription}>
  114            {iconDescription && <title>{iconDescription}</title>}
  115          </ChevronDownGlyph>

\\path\to\carbon\packages\react\src\components\ToggleSmall\ToggleSmall.js:
   53            input = el;
   54          }}
   55:         aria-label={ariaLabel}
   56          onKeyUp={evt => {
   57            if (match(evt, keys.ENTER)) {

\\path\to\carbon\packages\react\src\components\ToolbarSearch\ToolbarSearch.js:
  117              <Search16
  118                className={`${prefix}--search-magnifier`}
  119:               aria-label={labelText}
  120              />
  121            </button>

\\path\to\carbon\packages\react\src\components\TooltipDefinition\TooltipDefinition.js:
   45          className={directionClassName}
   46          role="tooltip"
   47:         aria-label={tooltipText}>
   48          <span className={`${prefix}--tooltip__caret`} />
   49          <p>{tooltipText}</p>

\\path\to\carbon\packages\react\src\components\TooltipIcon\TooltipIcon.js:
   32    return (
   33      <div {...rest} className={tooltipClassName}>
   34:       <button className={triggerClassName} aria-label={tooltipText}>
   35          {children}
   36        </button>

63 matches across 43 files
```
</details>